src/dask_dirac/_dirac.py

```diff
@@ -22,6 +22,10 @@ class DiracSettings:
 
 
 def _query(settings: DiracSettings, params: dict[str, str]) -> Any:
+
+    if "diracdev.grid.hep.ph.ic.ac.uk" in settings.query_url:
+        params["clientSetup"] = "GridPP"
+
```

Comment:
To avoid increasing the complexity of the function should more such things occur, may I suggest

```python
def _set_defaults(settings: DiracSettings, params: dict[str, str]):
    if "diracdev.grid.hep.ph.ic.ac.uk" in settings.query_url:
        params["clientSetup"] = params.get("clientSetup", "GridPP")
    return params

def _query(settings: DiracSettings, params: dict[str, str]) -> Any:
    params = _set_defaults(settings, params)
    ...
```